lighthouse-core/config/seo.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  extends: 'lighthouse:default',
```

Comment:
I had anticipated using the SEO config as a staging ground while the audits are developed. When stable, I planned to merge it with the default config. Happy to do whatever works best for you.